TrafficCapture/dockerSolution/README.md

```diff
@@ -14,6 +14,32 @@ While in the TrafficCapture directory, run the following command:
 
 `./gradlew :dockerSolution:composeUp`
 
+### Interacting with the Docker Solution
+
+A sample source cluster (running Elasticsearch 7.10) is set up with the Capture Proxy running in front of it. To route
+calls through the Capture Proxy, send requests to `localhost:9200` with the default credentials (admin, admin), e.g.:
+```sh
+curl --insecure https://localhost:9200/_cat/indices -u 'admin:admin'
+```
+
+You can send the same calls to the source cluster while bypassing the Capture Proxy (calls will not be relayed to the
+target cluster) via `localhost:19200`, and to the target cluster directly at `localhost:29200`.
+
+For sample data that exercises various endpoints with a range of datatypes, you can `ssh` into the Migration Console
+(`docker exec -i -t $(docker ps -aqf "ancestor=migrations/migration_console:latest") bash` or via the Docker console)
+and run `./runTestBenchmarks.sh`. By default, this runs four workloads from
+[Opensearch-Benchmark](https://github.com/opensearch-project/opensearch-benchmark) against the Capture Proxy endpoint
+(9200). The Migration Console contains other utility functions (`./catIndices.sh`, `kafka-tools`, etc.) to interact
+with the various containers of the solution.
+
+You can also access the metrics generated by the solution in Grafana. While the solution is running, go to
```

Comment:
I would prefix this with "with the default docker-compose" configuration launched with :dockerSolution:composeUp, instrumentation containers will be started (see below for other options).